fluent-langneg/test/locale_test.js

```diff
@@ -51,11 +51,11 @@ suite('Parses simple locales', () => {
       variant: 'mac'
     }));
 
-    assert.ok(isLocaleEqual('lij-Arab-FA-linux', {
+    assert.ok(isLocaleEqual('lij-Arab-FA-lin', {
```

Comment:
According to https://tools.ietf.org/html/bcp47#section-2.2.5, that's a bug. [a-z] variants must be at least 5 characters long.

We should fix that on our gecko impl, and just whitelist 'mac' in ja-JP-mac as an exception.